config/jail.conf

```diff
@@ -330,7 +347,7 @@ logpath = /opt/openhab/logs/request.log
 port    = http,https
 logpath = %(nginx_error_log)s
 
-# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module` 
+# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module`
```

Comment:
please don't touch lines not affected by this PR (I suspect you've trim ending spaces in your editor, but you can revert such lines before commit, so please rebase this)